README.md

```diff
@@ -162,7 +162,7 @@ Create remote data provider by calling `CompleterService.remote`.
 |imageField|string|Name of the field to use as image url for the list item.|
 |urlFormater|(term: string) => string|Function that get's the searchterm and returns the search url before each search.|
 |dataField|string|The field in the response that includes the data.|
-|headers|Headers (@angular/http)|HTTP request headers that should be sent with the search request.
```

Comment:
Sure - can easily keep both. I wonder if some sort of logic to determine whether the specified requestOptions should be sent, or just the specified headers e.g.:
```
/* 
 * If requestOptions are provided, they will override anything set in headers.
 *
 * If no requestOptions are provided, a new RequestOptions object will be instantiated,
 * and either the provided headers or a new Headers() object will be sent.
 */
if (!this._requestOptions) {
	this._requestOptions = new RequestOptions();
	this._requestOptions.headers = this._headers || new Headers();
}

this.remoteSearch = this.http.get(url, this._requestOptions)
...
```

This can be simplified once you eventually remove the headers attribute.

What do you think?